kitsu/script.js

```diff
@@ -0,0 +1,123 @@
+var API = 'https://nyaaapi.herokuapp.com/nyaa/anime?query='
+
+function insertAfter(referenceNode, newNode) {
+    referenceNode.parentNode.insertBefore(newNode, referenceNode.nextSibling);
+}
+
+var animeTitle = document.getElementsByTagName('h3')[0].innerText
+var summaryElement = document.getElementsByClassName('media-summary')[0]
+var torrentElement = document.createElement('div')
+
+var torrentsCount = 0
+var torrents = []
+var pageCount = 0
+
+function inflator(torrents) {
+    let string = `<h2>Torrents</h2><table class="table-recently-updated" width="100%" cellspacing="0" cellpadding="0" border="0"><tbody><select id="sorter">
+    <option value="time">Sort: Time - Ascending</option>
+    <option value="-time">Sort: Time - Descending</option>
+    <option value="seeders">Sort: Seeds - Ascending</option>
+    <option value="-seeders">Sort: Seeds - Descending</option>
+    <option value="size">Sort: File size - Ascending</option>
+    <option value="-size">Sort: File size - Descending</option>
+    </select> <button onclick="sort()">Sort</button>`
+    for (let i = (10 * pageCount); i < ((10 * pageCount) + endLimitDiscriminator()); i++) {
+        string += `<tr><td>
+    <div><a target="_blank" href="${torrents[i].link}">${torrents[i].title}</a>
+    <p style="float:right;">${torrents[i].size}</p></div>
+    <div><a style="float:left; margin-right: 5px;" target="_blank" href="${torrents[i].magnet}">Magnet</a>
+    <p style="float:left;">${torrents[i].seeders}↑</p><p style="float:left;">${torrents[i].leechers}↓</p>
+    <p style="float:right;">${torrents[i].time}</p></div></td></tr>`
+    }
+    string += `</tbody></table><center><p id="torrent-current-page">${pageCount + 1}/${pages()}</center><center><a onclick="pageBack();" style="margin-right: 5px; cursor: pointer;">Back</a><a onclick="pageNext();" style="cursor: pointer;">Next</a></center>`
```

Comment:
Try to make the UI better and more suited to Kitsu website UI. change the classes as well. This file is the exact copy of the file for MAL.

Reply:
I agree. Will do.